app/(student)/point-allocations/page.jsx

```diff
@@ -1,7 +1,311 @@
-import React from "react";
+"use client"
+import React, { useState } from 'react'
+import { BsThreeDotsVertical } from 'react-icons/bs';
+import {VscSettings} from 'react-icons/vsc'
+import {BiSolidReport} from 'react-icons/bi'
+import {AiFillEye, AiOutlineCaretDown, AiOutlineCaretUp} from 'react-icons/ai'
+import {BsXLg} from 'react-icons/bs'
+import {MdArrowDownward, MdArrowUpward} from 'react-icons/md'
 
-const PointAllocations = () => {
-  return <div>PointAllocations</div>;
-};
+ const PointAllocation = () => {
+  const initialIsOpenState = Array(data.length).fill(false);
+  const [isOpen, setIsOpen] = useState(initialIsOpenState);
```

Comment:
pwede mo naman na dito ipasa sa useState yung false mismo e haha pinahaba mo pa  code
```suggestion
  const [isOpen, setIsOpen] = useState(false);
```

Reply:
same dito pre, somehow magkacorrelate sila ng line 19 to 24 haha. may lumalabas na error kapag pinipindot ko yung threedots.

Reply:
ano nalabas na error?